apps/talon/talon_debug/talon_debug.py

```diff
@@ -0,0 +1,14 @@
+from talon import Module
```

Comment:
Potentially consider naming this talon_debug_window? It is a little confusing that it is an application that is not actually an application.